src/applications/disability-benefits/2346/components/oderSupplyPageContent.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I think this file should be named `orderSupplyPageContent`

Reply:
That's what it is named

Reply:
`order` is misspelled in the file name

Reply:
Yes it's corrected now thanks